agent/bench-scripts/pbench-user-benchmark

```diff
@@ -109,10 +109,10 @@ export benchmark_results_dir=$benchmark_results_dir
 # but since we don't call pbench-collect-sysinfo here, we need to call
 # metadata-log explicitly
 export benchmark config
-pbench-metadata-log --group=$tool_group --dir=$benchmark_run_dir beg
+pbench-metadata-log --group=$tool_group --dir=$benchmark_run_dir
 
 # on abnormal exit, make sure that the metadata log exists and is complete.
-trap "pbench-metadata-log --group=$tool_group --dir=$benchmark_run_dir int" INT QUIT
+trap "pbench-metadata-log --group=$tool_group --dir=$benchmark_run_dir" INT QUIT
```

Comment:
Restore the previous line, and then add one with "`end`" between lines 123 and 124.